src/report.js

```diff
@@ -35,29 +36,34 @@ module.exports = async function reportStatus(startTime, core, github, score) {
 
 Each problematic comment (i.e. comments with TODO, HACK or FIXME in it) contributes -${PROBLEMATIC_COMMENT_PENALTY} points to the health score.
 
-${score.comments.map((c) => `- \`${c.trim()}\``).join('\n')}\n`;
+${score.comments.map((c) => `- \`${c.comment}\``).join('\n')}\n`;
   }
   if (score.coverageMisses) {
     details += `\n## Code Coverage
 
 According to [the code coverage for this project](https://app.codecov.io/gh/${ctx.repo.owner}/${ctx.repo.repo}), there are ${score.coverageMisses} uncovered lines of code. Each uncovered line of code contributes -${UNCOVERED_LINE_PENALTY} to the health score.`;
   }
-
+  const annotations = getAnnotations(score.comments);
   // TODO: handle API call erroring out
-  await octokit.rest.checks.create({
-    name: 'Health Score',
-    owner: ctx.repo.owner,
-    repo: ctx.repo.repo,
-    head_sha: getSHA(core, github),
-    status: 'completed',
-    conclusion: 'neutral',
-    completed_at: new Date().toISOString(),
-    started_at: startTime.toISOString(),
-    output: {
-      title: `${points}`,
-      summary: `${points} health score points`,
-      text: details,
-    },
-  });
+  try {
+    await octokit.rest.checks.create({
+      name: 'Health Score',
+      owner: ctx.repo.owner,
+      repo: ctx.repo.repo,
+      head_sha: getSHA(core, github),
+      status: 'completed',
+      conclusion: 'neutral',
+      completed_at: new Date().toISOString(),
+      started_at: startTime.toISOString(),
+      output: {
+        title: `${points}`,
+        summary: `${points} health score points`,
+        text: details,
+        annotations,
+      },
+    });
+  } catch (e) {
+    core.error('Octokit checks creation call failed');
```

Comment:
Let's pass the `e` object into the `error` call in some form, so we know what error caused the call to fail.

Reply:
Done. Just added another `core.error` call